boxes/drawing.py

```diff
@@ -733,6 +734,17 @@ class LBRN2Surface(Surface):
         'monospaced' : 'Courier New'
     }
 
+    lbrn2_colors=[
+        0,  # Colors.OUTER_CUT    (BLACK)   --> Lightburn C00 (black)
```

Comment:
I wonder if inner and outer cut should be swapped color wise or if they should be re-ordered in the new CutSettings

Reply:
You are right - the table does the color matching and then there is a separate section where the cutting sequence is defined...